media-proxy/include/mesh/conn_rdma.h

```diff
@@ -0,0 +1,128 @@
+#ifndef CONN_RDMA_H
+#define CONN_RDMA_H
+
+#include "logger.h"
+#include "mesh/conn.h"
+#include "concurrency.h"
+#include "libfabric_ep.h"
+#include "libfabric_mr.h"
+#include "libfabric_cq.h"
+#include "libfabric_dev.h"
+#include "mcm_dp.h"
+#include <mutex>
+#include <cstring>
+#include <cstddef>
+#include <atomic>
+#include <queue>
+
+#ifndef RDMA_DEFAULT_TIMEOUT
+#define RDMA_DEFAULT_TIMEOUT 1
+#endif
+#ifndef MAX_BUFFER_SIZE
+#define MAX_BUFFER_SIZE 1 << 30
+#endif
+#ifndef CQ_BATCH_SIZE
+#define CQ_BATCH_SIZE 64
+#endif
+#ifndef PAGE_SIZE
+#define PAGE_SIZE 4096
+#endif
+
+namespace mesh {
+
+namespace connection {
```

Comment:
Change where applicable
```suggestion
namespace mesh::connection {
```